src/components/policy/policy_regular/include/policy/policy_table/enums.h

```diff
@@ -93,23 +94,23 @@ bool IsValidEnum(Parameter val);
 const char* EnumToJsonString(Parameter val);
 bool EnumFromJsonString(const std::string& literal, Parameter* result);
 
-enum AppHMIType {
-  AHT_DEFAULT,
-  AHT_COMMUNICATION,
-  AHT_MEDIA,
-  AHT_MESSAGING,
-  AHT_NAVIGATION,
-  AHT_INFORMATION,
-  AHT_SOCIAL,
-  AHT_BACKGROUND_PROCESS,
-  AHT_TESTING,
-  AHT_SYSTEM,
-  AHT_PROJECTION,
-  AHT_REMOTE_CONTROL
-};
-bool IsValidEnum(AppHMIType val);
-const char* EnumToJsonString(AppHMIType val);
-bool EnumFromJsonString(const std::string& literal, AppHMIType* result);
+// enum AppHMIType {
```

Comment:
why commented them out and left them in code, why not just delete them

Reply:
@yang1070 Fixed in d1308970ca2afad7de251f6ef744d94c4422cb42